format/index.md

```diff
@@ -266,45 +269,19 @@ consistently throughout an implementation.
 Each resource object **MUST** contain an `id` member, whose value **MUST**
 be a string.
 
-#### Links <a href="#document-structure-links" id="document-structure-links" class="headerlink"></a>
-
-The value of the `"links"` key is a JSON object (a "links object") that
-represents related resources, keyed by the name of each association.
-These associations share a namespace with [attributes]. Associations of a
-given resource object **MUST** be named differently than its [attributes].
-
-The keys `"id"`, `"type"` and `"self"` are reserved within the links object.
-
-#### Resource URLs <a href="#document-structure-resource-urls" id="document-structure-resource-urls" class="headerlink"></a>
-
-A resource object **MAY** include a URL in its links object, keyed by
-`"self"`, that identifies the resource represented by the resource object.
-
-```json
-// ...
-{
-  "type": "articles",
-  "id": "1",
-  "attributes": {
-    "title": "Rails is Omakase"
-  },
-  "links": {
-    "self": "http://example.com/articles/1"
-  }
-}
-// ...
-```
-
-A server **MUST** respond to a `GET` request to the specified URL with a
-response that includes the resource as the primary data.
+#### Relationships <a href="#document-structure-links" id="document-structure-resource-objects-relationships" class="headerlink"></a>
 
-#### Resource Relationships <a href="#document-structure-resource-relationships" id="document-structure-resource-relationships" class="headerlink"></a>
+The value of the `"relationships"` key is a JSON object (a "relationships object")
+that represents references from the resource in whose resource object it's defined
+to other resources ("relationships"). These relationships share a namespace with
+[attributes]; that is, relationships of a given resource object **MUST** be named
+differently than its [attributes].
 
-A resource object **MAY** contain references to other resource objects
-("relationships"). Relationships may be to-one or to-many. Relationships
-can be specified by including a member in a resource's [links] object.
+The keys `"id"` and `"type"` are not allowed within the relationships object.
 
-The name of the relationship declared in the key **SHALL NOT** be `"self"`.
+Relationships may be to-one or to-many. Relationships can be specified by
+including a member in a resource's relationship's object. The name of the
+relationship is its key in the the relationship object.
 
 The value of a relationship **MUST** be one of the following:
```

Comment:
@dgeb @tkellen How would you guys feel now about getting rid of the `"name": "related resource url"` representation of a relationship? There was an [argument for](https://github.com/json-api/json-api/pull/453) removing it before, and I think that argument's much stronger now, as this would be the only instance in the spec of a URL occuring outside of a `"links"` object.